VotingApplication/VotingApplication.Web/Scripts/Controllers/Create/ConfirmStepController.js

```diff
@@ -67,5 +83,73 @@
                 return choice.Name;
             });
         }
+
+        function getValidInvitees() {
+            return $scope.newPoll.Invitations.filter(function (invite) {
+                // Maybe this needs to be pulled out of this controller
+                var emailRegEx = /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i;
+                return emailRegEx.test(invite);
+            });
+        }
+
+        function formatUtcPollExpiry() {
```

Comment:
I am not sure this has anything to do with the display. This code filters the array of emails removing ones that aren't valid. This is used to generate the list of emails to send to the backend.